docs/CrossAccount.md

```diff
@@ -0,0 +1,89 @@
+# Cross-account Access
+
+It is often desirable to run the Quilt control plane (CloudFormation stack)
+in a separate account from your data plane (S3 Buckets).
+
+Assume that we have two accounts, *ControlAccount* (containing the Quilt
+CloudFormation stack) and *DataAccount* (containing the desired S3 buckets).
```

Comment:
Probably we should always use the same spelling `S3 Buckets` vs `S3 buckets`.

Reply:
Amazon uses "S3 buckets" (lowercase). https://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html